server/pulp/server/db/manage.py

```diff
@@ -191,6 +195,15 @@ def main():
         options = parse_args()
         _start_logging()
         connection.initialize(max_timeout=1)
+
+        # Prompt the user if there are workers that have not timed out
+        if filter(lambda worker: (UTCDateTimeField().to_python(datetime.now()) -
+                                  worker['last_heartbeat']).total_seconds() <
+                  constants.CELERY_TIMEOUT_SECONDS, status.get_workers()):
+            if not _user_input_continue('There are still running workers, continuing can '
+                                        'cause major data issues. Are you sure you wish '
```

Comment:
I like the language update. I was thinking about how to introduce an `corrupt` as an even scarier word. Consider s/can cause major data issues/could corrupt your Pulp installation/  I'm OK with it as is, so this is not a required change.